### PR TITLE
Switching to official useraccounts:core package

### PR DESCRIPTION
**Pull request by nathanathan**
This resolves the following story: https://www.pivotaltracker.com/n/projects/1369754/stories/97316502

### CODE REVIEW

File: .meteor/packages [hunk 1]
```diff
@@ -14,3 +14,4 @@ mantle:fixtures
 # CollectionFS security requires this to be global
 # https://github.com/CollectionFS/Meteor-CollectionFS/issues/711
 accounts-base
+useraccounts:core@1.12.2
```
Comment (aslagle) [inline .meteor/packages]: Is there some reason this needs to be in .meteor/packages? I'd probably put it in controllers, or some other package.

**Comment by aslagle** [comment]
Wow, it just worked? I was expecting we'd have to change something.

**Comment by nathanathan** [comment]
Yes, it worked without changes.

**Comment by aslagle** [comment]
The unit tests are fixed but now a couple cucumber tests are timing out.

**Comment by frosario** [comment]
All tests pass Circle CI now:
https://circleci.com/gh/ecohealthalliance/mantle/58

**Comment by aslagle** [comment]
What did you change?

**Comment by frosario** [comment]
I use meteor build before running the unit tests:
https://github.com/ecohealthalliance/mantle/blob/e675d652df28c0453265d25100408cbb9b44ea1d/continuous-integration.sh#L4

**Comment by aslagle** [comment]
Did you change something since my comment that a couple cucumber tests were timing out?

**Comment by frosario** [comment]
The only changes I made are part of commit: https://github.com/ecohealthalliance/mantle/commit/e675d652df28c0453265d25100408cbb9b44ea1d

**Comment by aslagle** [comment]
I saw a previous version of that commit that still failed, wondering why this one works.

**Comment by frosario** [comment]
That's the intermittent failure where an async script times out. Even after increasing the timeout to 30 seconds the job fails occasionally.

**Comment by aslagle** [comment]
Ah, I was hoping you had figured out how to fix that :)

**Comment by frosario** [comment]
Disabled phantomjs test, and it failed during the chrome run.
Async script timed out at this step:
`Then I should be on the "Test Organization" detail page`

**Comment by frosario** [comment]
Disabled phantomjs did not impact the async timeout issue at all. Reverted previous commit, and reenabled phantomjs cucumber testing.

**Pull request merged**